docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,44 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that reconciles BareMetalHost resources using information provided by Ironic and the Cluster API Provider Metal3 (CAPM3) controller. A simplified information flow path is shown below:
+
+```text
```

Comment:
I feel like this page should have more words. The example is great, but something should be said about BareMetalHost resources: what they represent, which fields they have, etc.

Reply:
Added an example and described the fields. 
I have made some broad fields grouping.